docs/index.rst

```diff
@@ -8,20 +8,20 @@ Fanatical Support for AWS Product Guide
 
 .. only:: latex
 
-  **IMPORTANT:** This is a PDF version of the Product Guide, and is intended
-  to be used for point-in-time offline reference purposes only. The
-  authoritative version of this document lives online at
-  `https://developer.rackspace.com/docs/fanatical-support-aws <https://developer.rackspace.com/docs/fanatical-support-aws>`_
-  and will contain the latest updates. |vspace|
+  **IMPORTANT:** Use this is PDF version of the Product Guide
+  for point-in-time offline reference purposes only. You can find the
+  authoritative version of this document with the latest updates at
+  `https://developer.rackspace.com/docs/fanatical-support-aws <https://developer.rackspace.com/docs/fanatical-support-aws>`_.
+  |vspace|
 
-This Product Guide is designed to provide a detailed look at how Rackspace
+This Product Guide provides a detailed look at how Rackspace
 delivers our **Fanatical Support for AWS** offering. It covers core concepts
 such as the :ref:`AWS account structure <aws_accounts>` and Rackspace
-:ref:`service levels <service_levels>`, and advanced concepts such as
+:ref:`service levels <service_levels>` and advanced concepts such as
 provisioning bastion access via :ref:`Rackspace Passport <passport_v2>` and
```

Comment:
Passport doesn't use bastion access anymore. Info here: https://docs.rackspace.com/docs/fanatical-support-aws/passport/

advanced concepts such as providing access requests to instances via Passport ...